test/ctia/entity/judgement_test.clj

```diff
@@ -50,11 +50,15 @@
         (is (= "Malicious" (first (map :disposition_name (:parsed-body response))))
             "IP quoted term works"))
 
-      (let [term "1.2.3.4"
-            response (GET app
-                          (str "ctia/judgement/search")
-                          :headers {"Authorization" "45c1f5e3f05d0"}
-                          :query-params {"query" term})]
+      ;; TODO: this test no longer works, because apparently, in order to
+      ;; perform search for values within nested fields (without Lucene
+      ;; syntax), we're going to have to implement a way po pass nested fields, e.g.,:
+      ;; "query_string": {"query": "1.2.3.4", "fields": ["observable.*^1.0"] - note the asterisk
```

Comment:
why `*`?
Assuming you are referring to the wildcard example [here](https://www.elastic.co/guide/en/elasticsearch/reference/current/query-dsl-query-string-query.html#query-string-multi-field), it should rather be ["observable.type",  "observable.value"].
So here, if your code properly works, the final query shall include default fields and for judgement `observable.value` shall be included.

Reply:
moreover the[ initial value](https://github.com/threatgrid/ctia/blob/70dd7d013a5401bc3c93f0c942e221828de57067/test/ctia/entity/judgement_test.clj#L44) for `term` is  `observable.value:\"1.2.3.4\"`
Why did you modify it?

Reply:
> Assuming you are referring to the wildcard example here, it should rather be ["observable.type", "observable.value"].

I agree. But that's what I'm saying. The logic in the current PR does not address nested fields (it doesn't even try to ascertain them). If you're saying we are at risk of introducing regression, then I better change it here, so it does recognize nested fields.

Reply:
99% of test which do not pass are regressions.
If you look at the test which does not pass, it's clearly a regression.